tests/unit/amo/api/test_collections.js

```diff
@@ -141,4 +179,200 @@ describe(__filename, () => {
       mockApi.verify();
     });
   });
+
+  describe('addAddonToCollection', () => {
+    const getAddParams = (params = {}) => {
+      return {
+        addon: 'addon-id-or-slug',
+        api: apiState,
+        collection: 'collection-id-or-slug',
+        user: 'user-id-or-username',
+        ...params,
+      };
+    };
+
+    it('posts an addon to a collection', async () => {
+      const params = getAddParams({
+        addon: 'some-addon',
+        user: 'my-username',
+        collection: 'my-collection',
+      });
+
+      mockApi
+        .expects('callApi')
+        .withArgs({
+          auth: true,
+          body: { addon: 'some-addon', notes: undefined },
+          endpoint:
+            'accounts/account/my-username/collections/my-collection/addons',
+          method: 'POST',
+          state: apiState,
+        })
+        .once()
+        .returns(createApiResponse());
+
+      await addAddonToCollection(params);
+      mockApi.verify();
+    });
+
+    it('posts notes about an addon', async () => {
```

Comment:
This spec confused me at first because I thought it was about posting notes but really it's about posting an add-on with accompanying notes, right? I think it would be clearer to label it as "posts an add-on and notes" or something like that.